beacon-chain/blockchain/service.go

```diff
@@ -92,6 +94,7 @@ func (c *ChainService) Start() {
 			if err := c.initializeBeaconChain(genesisTime, initialDeposits); err != nil {
 				log.Fatalf("Could not initialize beacon chain: %v", err)
 			}
+			c.stateInitializedFeed.Send(genesisTime)
```

Comment:
Where is this new logic being tested? I don't see any test assertions that a genesis time is sent from this routine

Reply:
Done